orchestration/dagster_orchestration/hca_manage/dataset.py

```diff
@@ -189,13 +196,13 @@ def create_dataset_with_policy_members(
         dataset_id: str = job_result["id"]
         logging.info(f"Dataset created, id = {dataset_id}")
 
-        # if policy-members are provided in the CLI args, add the monster team email (default without provided arg)
+        logging.info(f"Adding default policy_members {self.stewards}")
+        self.add_policy_members(dataset_id, self.stewards, "steward")
```

Comment:
This has been annoying me, we were attempting to always attach `monster@firecloud.org` in dev, which does not exist there. This cleans that up by attaching the correct stewards set by default and outputs more useful information after creating a dataset (in addition to the ID).